ui/src/server/handlers/api.js

```diff
@@ -61,6 +61,67 @@ Fetchr.registerService({
     },
 });
 
+Fetchr.registerService({
+    name: 'assertionConditions',
+    create(req, resource, params, body, config, callback) {
+        let assertionConditions = [];
+        let assertionConditionData = {
+            operator: 'EQUALS',
+            value: '',
+        };
+
+        var assertionCondition;
+
+        for (var i = 0; i < params.assertionConditions.length; i++) {
+            let condition = {};
+            assertionCondition = {
+                conditionsMap: {},
+            };
+            Object.keys(params.assertionConditions[i]).forEach((key) => {
+                let copyAssertionConditionData = JSON.parse(
+                    JSON.stringify(assertionConditionData)
+                );
+                copyAssertionConditionData['value'] =
+                    params.assertionConditions[i][key];
+                if (copyAssertionConditionData['value'] === '') {
+                    copyAssertionConditionData['value'] = 'ALL';
```

Comment:
we can use * here now. regex pattern on the server is updated to allow * as a first char